SonarQube.TeamBuild.PostProcessor/ArgumentProcessor.cs

```diff
@@ -107,11 +107,11 @@ private static bool AreParsedArgumentsValid(IAnalysisPropertyProvider provider,
         /// <summary>
         /// Determines whether the supplied property is accepted by the post-processor
         /// </summary>
-        public static bool IsPermittedProperty(Property property)
+        private static bool IsPermittedProperty(Property property)
         {
             // Currently the post-processor only accepts command line arguments that
             // will be stripped from teh the pre-processor command line
-            return ProcessRunnerArguments.SensitivePropertyKeys.Any(marker => Property.AreKeysEqual(marker, property.Id));
+            return ProcessRunnerArguments.ContainsSensitiveData(property.Id);
```

Comment:
The only reason to access the SensitivePropertyKeys directly is to use the equality helper. Using the same filter (ContainsSensitiveData on the key) will allow users to potentially pass in more properties to the post-processor which is consistent with the fact that these properties will not be serialized. In practice, I doubt this matters too much, so might as well have fewer dependencies.